src/r5py/r5/transport_network.py

```diff
@@ -41,15 +43,21 @@ def __init__(self, osm_pbf, gtfs=[], build_config={}):
         build_json : dict
             options accepted by TNBuilderConfig (including SpeedConfig)
         """
-        # TODO: Add TNBuilderConfig and SpeedConfig options to docstring
```

Comment:
I removed this TODO comment because it becomes obsolete with the next R5 version

Reply:
👍🏻